src/app/modules/item/task-communication/rxjschannel.ts

```diff
@@ -81,7 +85,8 @@ export class RxMessagingChannel {
     this.channel.notify(message);
```

Comment:
Shouldn't all call to `channel` (here, on line 80, 64, 37) make sure it is not destroyed first? (maybe with exceptions here)
From what I see it should not really happen if the class is well used, but so that's relying on the caller, so an extra checks looks useful.

Reply:
This method is unused, so I'd rather do nothing on it :sweat_smile:

Reply:
(But ok for other calls)

Reply:
fixed